front/app/components/UI/MentionsTextArea/index.tsx

```diff
@@ -127,6 +127,7 @@ const MentionsTextArea = ({
           border: 'none',
           appearance: 'none',
           WebkitAppearance: 'none',
+          minHeight: `${rows * parseInt(lineHeight as string, 10)}px`,
```

Comment:
```suggestion
          minHeight: `${rows * parseInt(lineHeight, 10)}px`,
```

Is already a string. Same on line 139.